scripts/deployment-helpers.sh

```
@@ -230,3 +343,235 @@ wait_for_csv() {
+  updateStrategy:
+    registryPoll:
+      interval: 10m
+EOF
+
+  echo "  * Waiting for CatalogSource to be ready..."
+  local timeout=120
+
+  if ! kubectl wait catalogsource "$name" -n "$namespace" \
+      --for=jsonpath='{.status.connectionState.lastObservedState}'=READY \
+      --timeout="${timeout}s" 2>/dev/null; then
+    local state=$(kubectl get catalogsource "$name" -n "$namespace" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)